lib/engine/config/game/g_1856.rb

```diff
@@ -517,18 +517,24 @@ module G1856
       "sym": "CGR",
       "logo": "1856/CGR",
       "name": "Canadian Government Railway",
-      "tokens": [
-        100,
-        100,
-        100,
-        100,
-        100,
-        100,
-        100,
-        100,
-        100
-      ],
-      "color": "cgrBlack"
+      "tokens": [],
+      "color": "cgrBlack",
+      "abilities": [
+         {
+            "type": "train_buy",
+            "description": "Inter train buy/sell at face value",
+            "face_value": true
+         },
+         {
+            "type": "train_limit",
+            "description": "+1 train limit",
+            "increase": 100
```

Comment:
By making the increase unreasonably big the goal is to prevent the forced train discards until after nationalization has completed

Reply:
I think you can modify description to something better. An alternative is to introduce a new attribute to cater for this case, but I suppose the solution is OK.

Reply:
After the CGR forms during nationalization it does actually have a +1 train limit. Are you suggesting I change it to say "3 train limit" instead of "+1 train limit"?